.github/workflows/rust.yml

```
@@ -139,3 +139,24 @@ jobs:
           $env:VCPKG_ROOT = $env:VCPKG_INSTALLATION_ROOT
           cargo build
           cargo test
+  windows-arm64:
+    runs-on: windows-latest
```

I've setup a runner for this:
```suggestion
    runs-on: windows-arm
```